driver/terraform.go

```diff
@@ -589,7 +589,7 @@ func (tf *Terraform) initTaskTemplate() error {
 
 // validateTemplate verifies that executing the fetch requests of
 // a template's dependencies does not error.
-func validateTemplate(t *hcat.Template, clients hcat.Looker) error {
+func validateTemplate(t templates.Template, clients hcat.Looker) error {
```

Comment:
Any specific reason why we switched from pass-by-reference to pass-by-value here?

Reply:
I believe `template.Templates` is an interface, and so the underlying object would be passed by reference